### PR TITLE
Test the pr nav

### PR DESCRIPTION
**Pull request by planteater**
check the boxes

### CODE REVIEW

File: freezer.js [hunk 1]
```diff
@@ -51,6 +51,14 @@ export class Container {
 		await this._agent.start();
 	}
 
+	private static onAboutSomething(e: ConfigurationWillChangeEvent) {
+		this._config = undefined;
+
+		if (configuration.changed(e.change, configuration.name("traceLevel").value)) {
+			Logger.level = configuration.get<TraceLevel>(configuration.name("traceLevel").value);
```
Comment (planteater) [inline freezer.js:58]: Add to a review, line 58

**Review by planteater — COMMENTED** [review]
all the reviewz

File: .idea/codestream.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    <option name="teamId" value="5f3d7746d26912027e8c36ef" />
+    <option name="webViewContext" value="{&quot;chatProviderAccess&quot;:&quot;strict&quot;,&quot;currentTeamId&quot;:&quot;5f3d7746d26912027e8c36ef&quot;,&quot;currentStreamId&quot;:&quot;&quot;,&quot;pullRequestCheckoutBranch&quot;:false,&quot;isRepositioning&quot;:false,&quot;panelStack&quot;:[&quot;status&quot;,&quot;sidebar&quot;,&quot;codemarks-for-file&quot;,&quot;landing-redirect&quot;],&quot;hasFocus&quot;:true,&quot;channelFilter&quot;:&quot;all&quot;,&quot;channelsMuteAll&quot;:false,&quot;codemarkFileFilter&quot;:&quot;all&quot;,&quot;codemarkTypeFilter&quot;:&quot;all&quot;,&quot;codemarkTagFilter&quot;:&quot;all&quot;,&quot;codemarkBranchFilter&quot;:&quot;all&quot;,&quot;codemarkAuthorFilter&quot;:&quot;all&quot;,&quot;codemarksFileViewStyle&quot;:&quot;inline&quot;,&quot;codemarksShowArchived&quot;:false,&quot;codemarksShowResolved&quot;:false,&quot;showFeedbackSmiley&quot;:true,&quot;route&quot;:{&quot;name&quot;:&quot;newUserEntry&quot;,&quot;params&quot;:{}},&quot;spatialViewShowPRComments&quot;:false,&quot;codemarksWrapComments&quot;:false}" />
   </component>
```
Comment (planteater) [inline .idea/codestream.xml:6]: just outside the range, start a rev

File: .idea/codestream.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CodeStream">
-    <option name="teamId" value="5c75994ba5969918d6f296e3" />
-    <option name="webViewContext" value="{&quot;chatProviderAccess&quot;:&quot;strict&quot;,&quot;currentTeamId&quot;:&quot;5c75994ba5969918d6f296e3&quot;,&quot;currentStreamId&quot;:&quot;&quot;,&quot;pullRequestCheckoutBranch&quot;:false,&quot;isRepositioning&quot;:false,&quot;panelStack&quot;:[&quot;codemarks-for-file&quot;,&quot;landing-redirect&quot;],&quot;hasFocus&quot;:true,&quot;channelFilter&quot;:&quot;all&quot;,&quot;channelsMuteAll&quot;:false,&quot;codemarkFileFilter&quot;:&quot;all&quot;,&quot;codemarkTypeFilter&quot;:&quot;all&quot;,&quot;codemarkTagFilter&quot;:&quot;all&quot;,&quot;codemarkBranchFilter&quot;:&quot;all&quot;,&quot;codemarkAuthorFilter&quot;:&quot;all&quot;,&quot;codemarksFileViewStyle&quot;:&quot;inline&quot;,&quot;codemarksShowArchived&quot;:false,&quot;codemarksShowResolved&quot;:false,&quot;showFeedbackSmiley&quot;:true,&quot;route&quot;:{&quot;name&quot;:&quot;newUserEntry&quot;,&quot;params&quot;:{}},&quot;spatialViewShowPRComments&quot;:false}" />
```
Comment (planteater) [inline .idea/codestream.xml:5]: one more add

File: foo.js [hunk 4]
```diff
@@ -23,6 +23,13 @@ export interface ApplyMarkerCommandArgs {
 	marker: CSMarkerIdentifier;
```
Comment (planteater) [inline foo.js:23]: rev next

**Review by planteater — COMMENTED** [review]
all unz

File: foo.js (hunk 4 above)
Comment (planteater) [reply foo.js:23]: rev rep

**Review by codestreamtest — APPROVED** [review]
lgtm?

**Review by planteater — COMMENTED** [review]
Review with comment submit

**Pull request closed without merging**